packages/fiori/src/UserMenu.ts

```diff
@@ -0,0 +1,359 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import {
+	customElement, slot, event, property,
+} from "@ui5/webcomponents-base/dist/decorators.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import DOMReferenceConverter from "@ui5/webcomponents-base/dist/converters/DOMReference.js";
+import Avatar from "@ui5/webcomponents/dist/Avatar.js";
+import Title from "@ui5/webcomponents/dist/Title.js";
+import Text from "@ui5/webcomponents/dist/Text.js";
+import Button from "@ui5/webcomponents/dist/Button.js";
+import Label from "@ui5/webcomponents/dist/Label.js";
+import Panel from "@ui5/webcomponents/dist/Panel.js";
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+import List, { type ListItemClickEventDetail } from "@ui5/webcomponents/dist/List.js";
+import ListItemCustom from "@ui5/webcomponents/dist/ListItemCustom.js";
+import Tag from "@ui5/webcomponents/dist/Tag.js";
+import ResponsivePopover from "@ui5/webcomponents/dist/ResponsivePopover.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import i18n from "@ui5/webcomponents-base/dist/decorators/i18n.js";
+import UserMenuAccount from "./UserMenuAccount.js";
+import UserMenuItem from "./UserMenuItem.js";
+import UserMenuTemplate from "./generated/templates/UserMenuTemplate.lit.js";
+import UserMenuCss from "./generated/themes/UserMenu.css.js";
+
+// Icons
+import "@ui5/webcomponents-icons/dist/add-employee.js";
+import "@ui5/webcomponents-icons/dist/person-placeholder.js";
+import "@ui5/webcomponents-icons/dist/log.js";
+import "@ui5/webcomponents-icons/dist/user-settings.js";
+
+// Texts
+import {
+	USER_MENU_OTHER_ACCOUNT_BUTTON_TXT,
+	USER_MENU_MANAGE_ACCOUNT_BUTTON_TXT,
+	USER_MENU_SIGN_OUT_BUTTON_TXT,
+	USER_MENU_POPOVER_ACCESSIBLE_NAME,
+	USER_MENU_EDIT_AVATAR_TXT,
+	USER_MENU_ADD_ACCOUNT_TXT,
+} from "./generated/i18n/i18n-defaults.js";
+
+type UserMenuItemClickEventDetail = {
+	item: UserMenuItem;
+}
+
+type UserMenuOtherAccountClickEventDetail = {
+	prevSelectedAccount: UserMenuAccount;
+	selectedAccount: UserMenuAccount;
+}
+
+/**
+ * @class
+ * ### Overview
+ *
+ * The `ui5-user-menu` is an SAP Fiori specific web component that is used in `ui5-shellbar`
+ * and allows the user to easily see information and settings for the current user and all other logged in accounts.
+ *
+ * ### ES6 Module Import
+ * `import "@ui5/webcomponents-fiori/dist/UserMenu.js";`
+ *
+ * `import "@ui5/webcomponents-fiori/dist/UserMenuItem.js";` (for `ui5-user-menu-item`)
```

Comment:
I see that UserMenuAccount is also imported, so it comes with the UserMenu as well.
Either describe it here, if expected, or we need to use it as a type and import it as a type if you don't want UserMenuAccount to come automatically with the UserMenu.

Reply:
I will leave it open for now to discuss it offline.